src/model/channel.ts

```diff
@@ -0,0 +1,21 @@
+// 全チャンネルタイプ
+export type ChannelType =
+  | 'Text' // ID: 0
+  | 'Voice' // ID: 2
+  | 'Category' // ID: 4
+  | 'Announce' // ID: 5
+  | 'Announce(Thread)' // ID: 10
+  | 'Thread(Public)' // ID: 11
+  | 'Thread(Private)' // ID: 12
+  | 'Stage' // ID: 13
+  | 'Forum'; // ID: 15
```

Comment:
```suggestion
  | 'Text'
  | 'Voice'
  | 'Category'
  | 'Announce'
  | 'Announce(Thread)'
  | 'Thread(Public)'
  | 'Thread(Private)'
  | 'Stage'
  | 'Forum';
```
こちらの定義は Discord API に依存しないので, ID の記載はやめてください